concrete/authentication/concrete/form.php

```diff
@@ -15,12 +16,14 @@
 		<input name="uPassword" id="uPassword" class="form-control" type="password" />
 	</div>
 
-	<div class="checkbox">
-		<label>
-			<input type="checkbox" name="uMaintainLogin" value="1">
-			<?= t('Stay signed in for two weeks') ?>
-		</label>
-	</div>
+    <?php if (Config::get('concrete.session.remember_me.lifetime') > 0) : ?>
+        <div class="checkbox">
+            <label>
+                <input type="checkbox" name="uMaintainLogin" value="1">
+                <?php echo t('Stay signed in for %s', $dh->describeInterval(Config::get('concrete.session.remember_me.lifetime'))); ?>
+            </label>
+        </div>
+    <?php endif; ?>
```

Comment:
Can you update this to use the standard brackets?

Reply:
@KorvinSzanto just for my future reference, is the alternate syntax not preferred for template files over using brackets? I find it quite easier to make sense of PHP code on a file with HTML with the alternate syntax because we can get a bunch of closing brackets that are not as easy to read. Anyway, I'm cool with updating if this is a Concrete5 requirement.

Reply:
@KorvinSzanto Done as requested.